host/sampi/http.go

```diff
@@ -95,22 +106,28 @@ func startJSONEventStreaming(w http.ResponseWriter) *json.Encoder {
 
 // HTTP Route Handles
 func (c *HTTPAPI) ListHosts(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	l := c.logger.New("fn", "ListHosts")
 	ret, err := c.Cluster.ListHosts()
 	if err != nil {
+		l.Error("error listing hosts", "err", err)
 		httphelper.Error(w, err)
 		return
 	}
+	l.Debug("writing http response")
```

Comment:
This is unncessary and should be handled by a log line automatically added at the end of every request (we should have a standard way of doing this automatically)